ext/storage_sources/azure_store/azure_connection.h

```diff
@@ -33,10 +33,20 @@
 
 #include <string>
 
+/*
+ * In Azure: Buckets are synonymous with Containers, and Objects are synonymous with Blobs. This
```

Comment:
Great comment!! I would just re-arrange it the first two sentences such that:
```
* This class represents an active connection to the Azure endpoint and allows for interaction with the
* Azure client. The Azure cloud stroage names buckets as containers and objects as blobs.
```

Reply:
done :)